### PR TITLE
a better ItemBuilder

### PR DESCRIPTION
**Pull request by noaottermann**
accept minemobs, accept...

### CODE REVIEW

**Comment by noaottermann** [comment]
bruh l'erreur elle vient pas de moi

**Review by Minemobs — CHANGES_REQUESTED** [review]
You should use a ternary operator instead

**Comment by noaottermann** [comment]
lamba go brrr just explain how to modify a pull request

**Comment by noaottermann** [comment]
* What went wrong:
1 actionable task: 1 executed
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not download spigot-1.17.1-R0.1-SNAPSHOT.jar (org.spigotmc:spigot:1.17.1-R0.1-SNAPSHOT:20210708.170[25](https://github.com/Minemobs/MinemobsUtils/runs/5613226018?check_suite_focus=true#step:5:25)3-1): Skipped due to earlier error

**Comment by Minemobs** [comment]
haha don't care

**Review by Minemobs — APPROVED** [review]

File: src/main/java/fr/minemobs/minemobsutils/utils/ItemBuilder.java [hunk 1]
```diff
@@ -156,9 +148,7 @@ public ItemBuilder addItemFlag(ItemFlag flag) {
     }
 
     public ItemBuilder addItemFlag(ItemFlag... flags) {
-        for (ItemFlag flag : flags) {
-            addItemFlag(flag);
-        }
+        Arrays.stream(flags).forEach(this::addItemFlag);
```
Comment (Minemobs) [inline src/main/java/fr/minemobs/minemobsutils/utils/ItemBuilder.java:152]:
```
/home/runner/work/MinemobsUtils/MinemobsUtils/src/main/java/fr/minemobs/minemobsutils/utils/ItemBuilder.java:151: error: cannot find symbol
        Arrays.stream(flags).forEach(this::addItemFlag);
        ^
  symbol:   variable Arrays
  location: class ItemBuilder

```

**Review by Minemobs — CHANGES_REQUESTED** [review]

**Review by Minemobs — APPROVED** [review]

**Pull request merged**